browser-test/src/admin/admin_question_preview.test.ts

```diff
@@ -191,7 +190,22 @@ test.describe('Admin question preview', {tag: ['@northstar']}, () => {
     })
   })
 
-  // TODO(#7849): Support file upload question previews
+  test('File upload preview', async ({page, adminQuestions}) => {
+    await test.step('Create question', async () => {
+      await adminQuestions.addFileUploadQuestion({
+        questionName: questionName,
+      })
+    })
+
+    await test.step('Expect preview renders properly', async () => {
+      await adminQuestions.gotoQuestionEditPage(questionName)
+
+      await validateScreenshot(
```

Comment:
**suggestion (non-blocking):** If you pass in false for the `fullPage` argument on the `validateScreenshot` method, it might remove the header from your screenshot.

Reply:
It didn't seem to have an effect